lib/app.js

```diff
@@ -8,10 +9,15 @@ const app = express();
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
+app.use( cors({
+    origin: ['http://localhost:5501', 'https://alchemy-shopping-front-end-demo.netlify.app', 'http://localhost:7890'],
```

Comment:
remember these are just the URLs of the front ends we'll interact with so we won't need the 7890 but also no harm in having it here.